SimpleBlockX4.md

```diff
@@ -0,0 +1,120 @@
+# Simple Block Assembly
+
+#### Description of Final Product
+
+  * Four cubes/bricks created in any general CAD software
+  * Explicitly created void volume and graveyard volume example in Cubit/Trelis.
+  * Model run through most general processes required in Cubit/Trelis.
+
+#### Goals of Example
+
+  * Introduce the workflow with an assembly that does not have any geometric issues.
+  * Work through important processes on an example that is easy to visualize and understand.
+
+## Procedure Overview
+
+### In Initial CAD program
+_Any CAD engine should work for this process as long as it can export to a
+format importable by SpaceClaim or Cubit/Trelis. The preferred format is ACIS (.sat)._
+  1. Create four solid bricks/cubes with edges of length 100cm that are separated by 50 cm
+    * In Solidworks/Catia you can either make an assembly from 4 cubes or one part

```

Comment:
I think this needs a blank line and/or more indenting - it doesn't render properly if you look at it on github